bundles/org.openhab.binding.sonos/src/main/resources/ESH-INF/i18n/sonos_de.properties

```diff
@@ -1,17 +1,17 @@
 # binding
 binding.sonos.name = Sonos Binding
-binding.sonos.description = Dieses Binding integriert das Home Sound System von Sonos. Durch dieses k�nnen Sonos Speaker und Gruppen gesteuert werden.
+binding.sonos.description = Dieses Binding integriert das Home Sound System von Sonos. Durch dieses können Sonos Speaker und Gruppen gesteuert werden.
```

Comment:
It looks like applying a [GitHub suggestion](https://help.github.com/en/articles/incorporating-feedback-in-your-pull-request#applying-suggested-changes) changes the file encoding to UTF-8 which is currently not very nice because our property for translations files require ISO-8859 encoding. We maybe should change that in the future. I already filed an issue for it (see https://github.com/openhab/openhab-core/issues/1012).

Reply:
Thanks for fixing this, it was indeed the Github editor that messed it up...